model_analyzer/config/generate/optuna_run_config_generator.py

```diff
@@ -670,8 +670,10 @@ def _create_default_perf_analyzer_config(
         default_perf_analyzer_config.update_config_from_profile_config(
             model_config.get_field("name"), self._config
         )
-        if self._config.is_request_rate_specified(
-            self._search_parameters[
+
```

Comment:
This is technically correct, but you can just call `self._search_parameters.get_parameter("request_rate")` instead

Reply:
This gave error but ```
if self._search_parameters[
                 model_config.get_field("name")
            ].get_parameter("request_rate"):
``` worked.  Pushing the change.